### PR TITLE
docs: add documentation about how to migrate V0 createSvgIcon

### PR DESCRIPTION
**Pull request by chpalac**
## Overview

Adding documentation regarding how to migrate V0 custom icons created with `createSvgIcon`

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/178c2be4bcd691f2cf5874f771236f2e6b8c1cd5'>178c2be4bcd691f2cf5874f771236f2e6b8c1cd5</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx [hunk 1]
```diff
@@ -3,7 +3,9 @@ import { IconCatalog } from './Components/IconCatalog/IconCatalog.tsx';
 
 <Meta title="Concepts/Migration/from v0/Icons" />
 
-# v0 - v9 Icon Catalog
+# Icons Migration
+
+## v0 - v9 Icon Catalog
```
Comment (ling1726) [inline apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx:8]: I would place the icon catalog as the last section

Comment (chpalac) [reply apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx]: Moved https://github.com/microsoft/fluentui/pull/25828/commits/12c0af7e66ee407db13bc9ac99047a2dd266e43c

File: apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx [hunk 2]
```diff
@@ -14,3 +16,53 @@ V0 can handle switching between outline and filled versions of an icon automatic
 to achieve the same effect in v9. You can read about these steps in [the official v9 Icon docs](?path=/docs/concepts-developer-icons-icons--page).
 
 <IconCatalog />
+
+## createSvgIcon
+
+V0 exports a `createSvgIcon` function that allows creating a custom icon such as:
+
+```jsx
+export const MyIcon = createSvgIcon({
```
[the remark this answers is not in the document]
Comment (chpalac) [reply apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx:13]: Added here: https://github.com/microsoft/fluentui/pull/25828/commits/2873abde0f41cfd6ebd6db674d5d5484b89022f5 

Check if it's enough

File: apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx [hunk 3]
```diff
@@ -3,7 +3,84 @@ import { IconCatalog } from './Components/IconCatalog/IconCatalog.tsx';
...
+And its usage would be like:
+
+```jsx
+// Usage Example
+
+// Default outlined icon
+<MyIcon />
+
+// Filled Icon
+<MyIcon filled />
```
Comment (ling1726) [inline apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx:78]: let's also add some docs here about sizing and how v0 differs from v9. I think users will always have to size the v9 icons after migration right?

Comment (chpalac) [reply apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx:78]: Added https://github.com/microsoft/fluentui/pull/25828/commits/1e15a0c7c40150aa775e75a38c45d42b2aae1d5a

File: apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx [hunk 4]
```diff
@@ -3,7 +3,98 @@ import { IconCatalog } from './Components/IconCatalog/IconCatalog.tsx';
...
+`createSvgIcon` from V0 will allow you to set a range of pre-defined size values (`small`,`smaller`,`smallest`,`medium`,`large`,`largest`,`larger`). e.g.:
+
+```jsx
+<MyIcon size="small" />
+```
+
+The V9 wrapIcon has a different approach allowing to set the fontSize which would change directly the icon size.
+
+```jsx
+<MyIcon fontSize={24} />
```
Comment (ling1726) [inline apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx:94]: not the only way, using css to style the `font-size` property also works :)

Comment (chpalac) [reply apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx:94]: @ling1726 added a mention to it.

**Review by ling1726 — APPROVED** [review]

**Pull request merged**